staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -223,6 +231,10 @@ type PodsMetricSource struct {
 	// targetAverageValue is the target value of the average of the
 	// metric across all relevant pods (as a quantity)
 	TargetAverageValue resource.Quantity `json:"targetAverageValue" protobuf:"bytes,2,name=targetAverageValue"`
+
+	// selector is the string-encoded form of a standard kubernetes label selector for the given metric
```

Comment:
Explain what selector means to the user in this context.  Why would they set it, what happens if it is empty, etc.

Reply:
Done (also in other areas this appears-- v2beta1 and v2beta2)